code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -17,6 +17,11 @@
 	var/limit = 10
 	var/speed = 1
 	var/list/holdingitems
+	///What objects will this allow for you to just insert and grind?
+	var/static/list/allowed_grindables = typecacheof(list(
+		/obj/item/food,
+		/obj/item/reagent_containers,
+		/obj/item/grown))
```

Comment:
Is this going to be a one time use, or should it be a define like IS_EDIBLE()

Reply:
This is rather unique in that this is just specifically things that can wind up in a plant bag or a serving tray for fast transfer. The benefit to doing a typecacheof here is that /obj/item/grown is explicitly non-edible, and even before the food refactor that's what caused the linked issue in the first place.